dist/windows/installer.nsi

```diff
@@ -32,27 +32,7 @@ Section $(inst_qbt_req) ;"qBittorrent (required)"
   File "qbittorrent.exe"
   File "qbittorrent.pdb"
   File "qt.conf"
-  File /oname=translations\qtbase_ar.qm "translations\qtbase_ar.qm"
-  File /oname=translations\qtbase_bg.qm "translations\qtbase_bg.qm"
-  File /oname=translations\qtbase_ca.qm "translations\qtbase_ca.qm"
-  File /oname=translations\qtbase_cs.qm "translations\qtbase_cs.qm"
-  File /oname=translations\qtbase_da.qm "translations\qtbase_da.qm"
-  File /oname=translations\qtbase_de.qm "translations\qtbase_de.qm"
-  File /oname=translations\qtbase_es.qm "translations\qtbase_es.qm"
-  File /oname=translations\qtbase_fi.qm "translations\qtbase_fi.qm"
-  File /oname=translations\qtbase_fr.qm "translations\qtbase_fr.qm"
-  File /oname=translations\qtbase_gd.qm "translations\qtbase_gd.qm"
-  File /oname=translations\qtbase_he.qm "translations\qtbase_he.qm"
-  File /oname=translations\qtbase_hu.qm "translations\qtbase_hu.qm"
-  File /oname=translations\qtbase_it.qm "translations\qtbase_it.qm"
-  File /oname=translations\qtbase_ja.qm "translations\qtbase_ja.qm"
-  File /oname=translations\qtbase_ko.qm "translations\qtbase_ko.qm"
-  File /oname=translations\qtbase_lv.qm "translations\qtbase_lv.qm"
-  File /oname=translations\qtbase_pl.qm "translations\qtbase_pl.qm"
-  File /oname=translations\qtbase_ru.qm "translations\qtbase_ru.qm"
-  File /oname=translations\qtbase_sk.qm "translations\qtbase_sk.qm"
-  File /oname=translations\qtbase_uk.qm "translations\qtbase_uk.qm"
-  File /oname=translations\qtbase_zh_TW.qm "translations\qtbase_zh_TW.qm"
+  File /r /x "qt_*.qm" "qtbase_*.qm"  ; omit translations folder path to preserve folder structure
```

Comment:
Please explain the changes in this file.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/12283/files#r398337620